activejob/lib/active_job/exceptions.rb

```diff
@@ -7,6 +7,11 @@ module ActiveJob
   module Exceptions
     extend ActiveSupport::Concern
 
+    included do
+      class_attribute :default_retry_jitter, instance_accessor: false, instance_predicate: false
+      self.default_retry_jitter = 0.15
```

Comment:
For next time, we generally refrain from prefixing configs with `default_`, that's implied.

Reply:
Let's rename. @kaspth Could you do a followup to chop `default_`?

Reply:
Done in https://github.com/rails/rails/commit/5ded839cfbfdb81dd1ad5a52c7dd3ef5874d3b2e

Reply:
@kaspth Thank you for the followup! I **think** I took the `default_` cue from somewhere else in the codebase but I’m not sure about that. In any case, thanks for mentioning it and thanks for correcting it. I was going to offer but since it’s already done... 🎉